wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationService.swift

```diff
@@ -267,13 +256,26 @@ public final class ConversationService: ConversationServiceInterface {
         )
 
         action.perform(in: context.notificationContext) { result in
+
             self.context.perform {
                 switch result {
-                case .success(let objectID):
-                    if let conversation = try? self.context.existingObject(with: objectID) as? ZMConversation {
-                        completion(.success(conversation))
-                    } else {
-                        completion(.failure(.conversationNotFound))
+                case .success(let response):
+                    Task {
```

Comment:
Let's make `internalCreateGroupConversation()` async and avoid this `Task`? Actions can also be performed async.

Reply:
@netbe you didn't address this

Reply:
I was trying not to change too much here, let's see how hard it gets

Reply:
I would do in a second PR